src/app/components/itenary/itenary.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, inject } from '@angular/core';
+import { ItenaryService } from '../../services/itenary.service';
+import { ItenaryFirebaseService } from '../../services/itenary-firebase.service';
+
+@Component({
+  selector: 'app-itenary',
+  standalone: true,
+  imports: [],
+  templateUrl: './itenary.component.html',
+  styleUrl: './itenary.component.css'
+})
+export class ItenaryComponent implements OnInit {
+  itenaryService = inject(ItenaryService);
+  itenaryFirebaseService = inject(ItenaryFirebaseService);
+
+
+  ngOnInit(): void {
+    this.itenaryFirebaseService.getItenaries().subscribe((itenaries) => {
```

Comment:
just remember to unsubscribe in ngOnDestroy, but this will likely change after you implement NgRx. Just in case it doesn't, check for necessary unsubscribes

Reply:
or use the self-destroy functionality

Reply:
👍